src/packages/user/user.controller.ts

```diff
@@ -0,0 +1,11 @@
+import { User, UserRepository } from './user.types';
+import { Controller } from '@nestjs/common';
+
+@Controller('User')
+export class UserController {
+    constructor(readonly repository: UserRepository) {}
+
+    async getCurrentUser(authHeader: string): Promise<User> {
+        return await this.repository.getCurrentUser(authHeader);
```

Comment:
getCurrentUser can throw. Is this being handled somewhere?

Reply:
I believe (and please correct me if i am wrong) that nest will catch this and log it for us

Reply:
I've not used Nest but that seems to be correct as per https://docs.nestjs.com/exception-filters